### ISSUE TITLE
Protobuf 3 Swift 3 Code-gen issue with reserved words

### ISSUE THREAD

**Issue by gmckenzi** [alexeyxo/protobuf-swift#170]
I'm using the latest (this morning) version of the ProtoBuf3.0-Swift3.0 branch.
I'm finding an issue that didn't happen with the previous code around code generation when a protobuf message uses Swift keywords as properties.

This is a client project so I can't attach the proto file, but here's the issue: the message definition has properties with names like "static" and "required". The Swift code-gen worked previously, but now it is generating code that looks like this:

```
      if hasStatic {
          if let varSize`static` = `static`?.computeMessageSize(fieldNumber: 9) {
              serialize_size += varSizestatic
          }
      }
      if hasRequired {
          if let varSize`required` = `required`?.computeMessageSize(fieldNumber: 10) {
              serialize_size += varSizerequired
          }
      }

```

Notice the backticks in the middle of the "varSizeStatic" where it is attempting to escape the reserved word but doing so in the middle of the Swift name causes an error.
Whereas this would have been more correct:

```
      if hasStatic {
          if let `varSizestatic` = `static`?.computeMessageSize(fieldNumber: 9) {
              serialize_size += varSizestatic
          }
      }
      if hasRequired {
          if let `varSizerequired` = `required`?.computeMessageSize(fieldNumber: 10) {
              serialize_size += varSizerequired
          }
      }
```

**Comment by DouglasHeriot** [comment]
I fixed this in pull request #164 , which @alexeyxo then based his changes on. I am not on the latest code yet, so am not sure how it ended up.

**Comment by alexeyxo** [comment]
Done.
